include/mqtt/broker/broker.hpp

```diff
@@ -2102,12 +2155,19 @@ class broker_t {
         publish_options pubopts,
         v5::properties props
     ) {
+        // Get auth rights for this topic
```

Comment:
Please add comment something like as follows:
auth_users prepared once here, and then referred multiple times in `subs_map_.modify()`.

Reply:
I often think 

```cpp
        auto auth_users = security.auth_sub(topic);
```

can be moved into deliver(). But it is inefficient. The comment help understanding.